gpu/test_gpu.py

```diff
@@ -70,6 +75,10 @@ def test_install_gpu_without_agent(self, configuration, machine_suffixes,
   def test_install_gpu_with_agent(self, configuration, machine_suffixes,
                                   master_accelerator, worker_accelerator,
                                   driver_provider):
+    # GPU agent supported on Dataproc 1.4+
+    if self.getImageVersion() < pkg_resources.parse_version("1.4"):
+      return
```

Comment:
Use skipTest instead (see other tests for an example)

Reply:
ack, fixed